common/src/main/java/org/apache/cassandra/sidecar/common/data/TokenRangeReplicasResponse.java

```diff
@@ -154,4 +155,74 @@ public String toString()
                    '}';
         }
     }
+
+    /**
+     * Class representing metadata associated with replica instances
+     */
+    public static class ReplicaMetadata
+    {
+        private final String state;
+        private final String status;
+        private final String name;
+        private final String address;
+        public ReplicaMetadata(@JsonProperty("state") String state,
+                               @JsonProperty("status") String status,
+                               @JsonProperty("name") String name,
+                               @JsonProperty("address") String address)
+        {
+            this.state = state;
+            this.status = status;
+            this.name = name;
+            this.address = address;
+        }
+
+        /**
+         * @return the node state. eg. NORMAL, JOINING, LEAVING, etc.
+         */
+        @JsonProperty("state")
+        public String state()
+        {
+            return state;
+        }
+
+        /**
+         * @return the node status. eg. UP, DOWN
+         */
+        @JsonProperty("status")
+        public String status()
+        {
+            return status;
+        }
+
+        /**
+         * @return FQDN of the node
+         */
+        @JsonProperty("name")
+        public String name()
```

Comment:
this should just be called `fqdn`, see https://github.com/apache/cassandra-sidecar/blob/trunk/common/src/main/java/org/apache/cassandra/sidecar/common/data/RingEntry.java#L171

Reply:
Addressed